### PR TITLE
Remove SameSite=None restrictions for Rack 2.1.0

### PR DESCRIPTION
**Pull request by kaspth**
Also bump Rails' now required Rack 2.1.0 version.

Pending Rack 2.1.0 release. cc @Jeremy @ioquatix

### CODE REVIEW

File: actionpack/actionpack.gemspec [hunk 1]
```diff
@@ -34,7 +34,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "activesupport", version
 
-  s.add_dependency "rack",      "~> 2.0"
+  s.add_dependency "rack",      "~> 2.0", ">= 2.1.0"
```
Comment (rafaelfranca) [inline actionpack/actionpack.gemspec]: I merged that change on 2-0-stable as well so we don't need to force 2.1. I'll release that version so we can remove this.

**Comment by vfonic** [comment]
Any chance this can get merged in? Chrome 80 is coming out on Feb 4th, in 8 days.

My app will start working without this fix. It doesn't make sense to manually have to patch it.

**Comment by yushun** [comment]
Chrome 80 is out. Is there any update to this or a workaround?

**Comment by vfonic** [comment]
Seems like Chrome team postponed this change for couple of weeks. I can't find the link now, but there should be something on their website.

**Pull request merged**